corehq/apps/app_manager/remote_app.py

```diff
@@ -81,9 +81,6 @@ def make_remote_profile(app):
                     download_index_url=download_index_url
                 )
 
-        if app.build_langs:
-            profile_xml.set_property("cur_locale", app.build_langs[0])
-
```

Comment:
This looks like it just removes functionality that was added presumably for a particular project that needed it. (If that's the case), did you go through someone to make sure that's no longer needed?